models/lung_nodule_ct_detection/configs/inference.json

```diff
@@ -135,14 +135,8 @@
         "collate_fn": "$monai.data.utils.no_collation"
     },
     "inferer": {
-        "_target_": "SlidingWindowInferer",
-        "roi_size": [
-            240,
```

Comment:
Hi @Can-Zhao , I cannot understand this line. With the change, how to adjust the roi size of the inferer?
According to this line (https://github.com/Project-MONAI/model-zoo/blob/6a893936088192810f6f8d8f30e5fce2e8faef4a/models/lung_nodule_ct_detection/scripts/detection_inferer.py#L38), it depends on the `detector`, but it has the patch size `512, 512, 192` (https://github.com/Project-MONAI/model-zoo/pull/267/files#diff-10c2358610640d3b97968476b3699b54d3433c6d17689a196a8a6a3fcb844e39R16)

Reply:
Thanks! In this case, 512, 512, 192 will be the self.detector.inferer.roi_size